async-nats/src/jetstream/stream.rs

```diff
@@ -1089,11 +1089,11 @@ impl TryFrom<RawMessage> for crate::Message {
     type Error = Error;
 
     fn try_from(value: RawMessage) -> Result<Self, Self::Error> {
-        let decoded_payload = base64::decode(value.payload)
+        let decoded_payload = crate::base64::decode_url(value.payload)
             .map_err(|err| Box::new(std::io::Error::new(ErrorKind::Other, err)))?;
         let decoded_headers = value
             .headers
-            .map(base64::decode)
+            .map(crate::base64::decode_url)
```

Comment:
Ah I missed this from the tests getting stuck on my system. This uses the normal dictionary, not the url base64 dictionary

Reply:
Should now be fixed